docs/core/install/localized-intellisense.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Install localized IntelliSense files
+description: Learn how to set up your development machine to use localized IntelliSense files.
+author: mairaw
+ms.author: mairaw
+ms.date: 12/12/2019
+---
+# How to install localized IntelliSense files for .NET Core projects
```

Comment:
You could probably drop the "projects" part of the title. These intellisense files could really be used for anything by anything. The user is installing them for the .NET Core product itself, not for projects. Otherwise, this may lead someone to think that if they want to "install intellisense for their own project" they follow these same steps.